kcidb/test_db.py

```diff
@@ -18,6 +19,17 @@ def test_schemas_main():
     assert_executes("", *argv,
                     stdout_re=r"4\.0: 4\.0\n4\.1: 4\.1\n")
 
+def test_reset(clean_database):
+    """Check kcidb-db-reset  works"""
```

Comment:
There's no such command-line tool, and it wouldn't make sense to create one, because `reset()` only affects the client-side state. This is a test for the `Client.reset()` method you just added. Please fix the docstring.